test/README.md

```diff
@@ -40,6 +40,29 @@ TEST(<your_test_suite_name>, <test_name>) {
 }
 ```
 
+## VSCode setting
+
+Although Googletest is automatically prepared, you may need to install it locally to highlight structs and functions.
+
+The below is an example of `c_cpp_properties.json` for Mac (assume Googletest is installed by brew).
+
+```json
+{
+    "configurations": [
+        {
+            ...
+            "includePath": [
+                "${workspaceFolder}/**",
+                "${workspaceFolder}/src",
+                "${workspaceFolder}/test",
+                "/opt/homebrew/include/"
```

Comment:
On CLion it is picked up even if it comes from `FetchContent`. It only needs the appropriate `#include` and then it can navigate the code. Is vscode really that flaky?

Anyway add `cmake-build-*/_deps` to use any `FetchContent` dependencies.

Reply:
@lan496, I'm using VSCode, too. With this setting, what will happen?

Reply:
Without `c_cpp_properties.json` setting, I see the following message:
![image](https://github.com/spglib/spglib/assets/7272505/1b4fe09d-8792-4d03-b8e0-b95acea1aae9)

Reply:
When I add `${workspaceFolder}/cmake-build-release-llvm/_deps/google-test-src/googletest/include"` to `includePath`, the above error message disappears.

Reply:
Doesn't VSCode have system/user/project level settings. It should be documented that `/opt/homebrew/include/` be set for the user space, while `${workspaceFolder}/cmake-build*/_deps` are in the project workspace.

But I find it weird that VSCode or a cmake extension does not use `compile_commands.json`. That is the standard way that cmake has been developing in order for the IDEs to use, and that's part of how CLion does not need special configurations.

Reply:
@LecrisUT, do you assume `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON`?

Reply:
No, the IDE automatically does it in its own temporary directory under the configured build directory. It also uses [`cmake-file-api`](https://cmake.org/cmake/help/latest/manual/cmake-file-api.7.html#manual:cmake-file-api(7)) to complement and bootstrap the process. You can generally see if the IDE is using that in `${build_dir}/.cmake/api/v1/{query,reply}`.

But in general this is completely transparent to the user and they don't need to setup any such custom variables.